### ISSUE TITLE
Lite wallet GUI won't show all offers and can't get offer identifier[Bug] 

### ISSUE THREAD

**Issue by DrKiz** [Chia-Network/chia-blockchain#10137]
### What happened?

Buggy lite wallet has locked up my XCH and a few CAT coins due to offer's I cannot cancel.

Cannot cancel in GUI as no offers after 49 show up.

Cannot cancel in CLI as I cannot figure out how to get the offer identifier -id from the offer file.

How do you cancel an offer in CLI if you can't parse the offer identifier (-id)?

Can't cancel it in GUI because no file shows up.

There needs to be a "cancel ALL outstanding offers" option.

tl/dr total fund lockup due to uncancellable offer 

### Version

chia wallet beta 1.2.11265

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by DrKiz** [comment]
Just an added comment.  You can  see the "offer identifier" in the GUI in the wallet for they key the offer was created with the 3 dots usually, but if the offer doesn't show up there... then there seems no way to get that -id string to use in CLI.

**Comment by esaung** [comment]
This has been fixed and will be released in an upcoming update - https://github.com/Chia-Network/chia-blockchain/issues/9936.

The mitigation for this is to send a transaction to yourself for the amount (or close to) that you had in the offers. this would effectively spend the coin that was locked up and prevent anyone else from accepting your offer if you shared it off your computer. if you need further help with this, please start a discussion in the support section.
